packages/markdown/src/markdown.spec.tsx

```diff
@@ -0,0 +1,24 @@
+import { render } from "@react-email/render";
+import { Markdown } from "./markdown";
+
+describe("ReactEmailMarkdown component renders correctly", () => {
+  it("renders the markdown in the correct format for browsers", () => {
+    const actualOutput = render(
```

Comment:
Can we add more tests cases? heading, text, link and list

Reply:
On it!.